### ISSUE TITLE
displayOptions not work in graphql-voyager/middleware

### ISSUE THREAD

**Issue by fangjj** [APIs-guru/graphql-voyager#35]
displayOptions not work in graphql-voyager/middleware.
```.javascript
router.all('/voyager', voyagerMiddleware({
  endpointUrl: '/graphql',
  displayOptions: {
    sortByAlphabet: true
  }
}));
```

**Comment by IvanGoncharov** [comment]
@fangjj Implemented in `v1.0.0-rc.8`, can you please test it?

**Comment by fangjj** [comment]
nice, i‘m willing to test it.
